grafast/grafserv/src/middleware/graphql.ts

```diff
@@ -565,3 +565,67 @@ export const makeGraphQLHandler = (
     }
   };
 };
+
+export const makeGraphQLHandler = (
```

Comment:
Correct me if I'm wrong, but `handleGraphQLHandlerError` must be catched on `graphqlHandler` rejection ? not on it's factory ??

Reply:
Uh, yeah... Ooops :)

Reply:
Kool, that's what I did. But maybe there's a better way to do it...